### DISCUSSION TITLE
Consider multichannel out

### DISCUSSION THREAD

**Discussion by overtinkering** [shorepine/tulipcc#320] [General]
Please consider at least  dual stereo out. It will allow a Master OUT(stereo) for the audience and Monitor OUT(headphones!) for CUEing/ monitoring audio before mixing it into the Master OUT. Like in a DJ performance workflow.
It should be doable on TDM or dual I2S, which is available on the ESP32 (not completely sure if on the S3)

**Comment by bwhitman** [comment]
Certainly possible. I'd recommend [prototyping a breadboard tulip or etc](https://github.com/shorepine/tulipcc/blob/main/docs/tulip_build.md) and wiring up a multichannel i2s setup there first. It would be more complicated to modify an extra out on the shipping Tulip CC.

**Comment by overtinkering** [comment]
if there is access to the second I2S on the main board on a ADC/DAC "expansion" port a daughterboard/ module could add 2 or more(through TDM) audio outs xor inputs.

**Comment by richmanaust** [comment]
That would be handy. I have it working as 4 seperate synth channels now with ability for drums as well (albeit this is buggy).  Each channel has it's effects bus. All that through a mono output is a bit lame.
